package.json

```diff
@@ -54,6 +54,18 @@
 						"if (typeof expr === \"undefined\")"
 					]
 				},
+				"postfix.snippetPreviewMode": {
+					"type": "string",
+					"enum": [
+						"raw",
+						"inserted"
+					],
+					"default": "raw",
```

Comment:
I think we discussed it should default to `inserted` in this case, right? It's more in line with what built-in snippets look like.